modules/persistent-storage-csi-gcp-hyperdisk-create-sc-during-install.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-hyperdisk-create-sc-during-install_{context}"]
+= Creating the storage class for C3 and N4 instance types
+
+If you are using hyperdisk-balanced disks, you need to create a specific storage class for hyperdisks rather than using the default storage class.
+
+.Prerequisites
+* Access to the cluster with administrative privileges.
+
+.Procedure
+To create a hyperdisk storage class during installation:
+
+. Follow the procedure in the _Installing a cluster on GCP with customizations_ section.
++ 
+For your install-config.yaml file, use the following example file:
++
+.Example install-config YAML file
+[source, yaml]
+----
+apiVersion: v1
+metadata:
+  name: ci-op-9976b7t2-8aa6b
+
+sshKey: |
+  XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+baseDomain: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+platform:
+  gcp:
+    projectID: XXXXXXXXXXXXXXXXXXXXXX
+    region: us-central1
+controlPlane:
+  architecture: amd64
+  name: master
+  platform:
+    gcp:
+      type: e2-custom-6-16384
+      osDisk:
+        diskType: pd-ssd
```

Comment:
If we write above that all machines should be able to handle hyperdisk-balanced, this one does not.

`type: n4-standard-4` and `diskType: hyperdisk-balanced` could be a good option